index.html

```diff
@@ -1052,6 +1097,160 @@ <h4 id="Abilities">Abilities</h4>
+            <tr>
+               <td>
+                  Legendary Empowered Ability
+               </td>
+               <td>
+                  <p>
+                  Upgrade to Empowered Ability. When you use Empowered Ability, the number
+                  of resistance points spent grants additional advantage (+) equal to the number of resistance points spent, 
+                  or grants additional damage equal to the number of resistance points spent - your choice. The type of damage 
+                  is the same as what the attack deals. If the ability deals no damage it can still deal this bonus damage of the 
+                  most appropriate type - GM's discretion.
+               </td>
+               <td>Can be taken once.
+               </td>
+            </tr>
+            <tr>
+               <td>
+                  Legendary Heroic Resistance
+               </td>
+               <td>
+                  <p>
+                  Upgrade to Heroic Resistance. On your turn for 0 AP you may spend any number of resistance points of any type.
+                  Until the start of your next turn you may use the Heroic Resistance ability a number of times equal to the 
+                  number of resistance points spent for no additional cost. Even if you don't use all of the Heroic Resistances
+                  you gained from spending resistance points in this way, you do not regain any resistance points spent.
+               </td>
+               <td>Can be taken once.
+               </td>
+            </tr>
+            <tr>
+               <td>Practised 
+               </td>
+               <td>
+                  Gain 1 additional use of each ability 
+               </td>
+               <td>Can be taken once.
+               </td>
+            </tr>
+            <tr>
+               <td>Restorative
+               </td>
+               <td>
+                  Heal an additional Light HP or gain additional Light Temp HP whenever you are healed or gain HP respectively.
+
+                  This uses your Light value.
+               </td>
+               <td>Can be taken once.
+               </td>
+            </tr>
+            <tr>
+               <td>Fast footed
+               </td>
+               <td>
+                  +1 AC when wearing any Armour and remove -5ft of the speed buff.
+               </td>
+               <td>Can be taken once.
+               </td>
+            </tr>
+            <tr>
+               <td>
+                  Unimpeded
+               </td>
+               <td>
+                  +1 AC when wearing Heavy Armour and remove -5ft of the speed buff.
+               </td>
+               <td>Can be taken once.
+               </td>
+            </tr>
+            <tr>
+               <td>
+                  Bull's Will
+               </td>
+               <td>
+                  Gain 5 Will Resistance Points.
+               </td>
+               <td>Can be taken once.
```

Comment:
Not really sure why these need harsh limits, they really don't feel very powerful and with all the other ways to regen / gain additional resistance points.